src/SupabaseClient.ts

```diff
@@ -154,9 +155,10 @@ export default class SupabaseClient {
     })
   }
 
-  private _initRealtimeClient() {
+  private _initRealtimeClient(options?: RealtimeClientOptions) {
     return new RealtimeClient(this.realtimeUrl, {
-      params: { apikey: this.supabaseKey },
+      ...options,
+      params: { apikey: this.supabaseKey, ...options?.params },
```

Comment:
@quicksnap let's flip the order so `apikey` is set later to bar users from overwriting via realtime params

Reply:
Done!